CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to react-native-rate-modal
+Contributing to react-native-rate-modal isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help make react-native-rate-modal better.
+
+## Working on react-native-rate-modal
+### Prerequisites
+* [Git](https://git-scm.com/)
+* Latest version of [Node.js](https://nodejs.org/en/)
+
+### Getting react-native-rate-modal
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/react-native-rate-modal.git
```

Comment:
Shouldn't it be my profile name? "omergulen" instead of "YOUR-GITHUB-PROFILE-NAME"

Reply:
no, as they fork it before creating pull request, they work on their repo's copy

Reply:
Yeah, I saw the 10th line just now. Thanks, if assignee of the issue won't respond in some time, I will merge your PR.